### PR TITLE
SOLR-16466: Admin UI - Make it optional to sort list of commandline args

### PR DESCRIPTION
**Pull request by freedev**
https://issues.apache.org/jira/browse/SOLR-16466

# Description

Added a call to action (checkbox) at end of JVM Command-Line Arguments. 
The arguments are unsorted (original order) by default , when the checkbox is selected the arguments are sorted lexicographically.

# Solution

Added a checkbox in the dashboard at end of JVM Command-Line Arguments. 
Changed Angular IndexController (the component behind the Admin Dashboard):
- changed reload function: save a copy of the original commandline args and check the order
- Added toggleCommandLineOrder function - handle the checkbox click
- Added showCommandLineArgs function - show the args list with the correct order 

# Tests

I have tested the changes locally running solr standalone

# Checklist

Please review the following and check all that apply:

- [X] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [X] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [X] I have developed this patch against the `main` branch.
- [X] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

**Review by elyograg — APPROVED** [review]
I admit I am a novice at web app design.  Nothing looks bad to me in the patch.

File: solr/CHANGES.txt [hunk 1]
```diff
@@ -8,6 +8,8 @@ https://github.com/apache/solr/blob/main/solr/solr-ref-guide/modules/upgrade-not
 
 New Features
 ---------------------
+* SOLR-16466: Admin UI - Make it optional to sort list of commandline args (Shawn Heisey, Vincenzo D'Amore via Christine Poerschke)
```
Comment (cpoerschke) [inline solr/CHANGES.txt:11]:
```suggestion
* SOLR-16466: Admin UI - Make it optional to sort list of commandline args (Shawn Heisey, Vincenzo D'Amore)
```

I didn't do much here really.

Comment (freedev) [reply solr/CHANGES.txt:11]: I don't agree you pointed me straight to the line where the problem was. All I had to do was write a more flexible implementation and test it.

**Comment by cpoerschke** [comment]
Hello again!

I'd love to see this included in the upcoming 9.6 release -- https://lists.apache.org/thread/mn6625txw938l8sy1o9r9rhmnqpdfo6v

@elyograg - would you have bandwidth to merge and cherry-pick/backport to `branch_9x` here or would it be more convenient for others to jump in at this stage for that?

cc/FYI @gus-asf as 9.6 release manager.

Thanks!

**Comment by elyograg** [comment]
I'm tied up today. If it should happen now, someone else can look into it.

**Pull request merged**